azure-mgmt-monitor/src/main/java/com/microsoft/azure/management/monitor/MetricDefinition.java

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ */
+
+package com.microsoft.azure.management.monitor;
+
+import com.microsoft.azure.management.apigeneration.LangMethodDefinition;
+import org.joda.time.DateTime;
+import org.joda.time.Period;
+import rx.Observable;
+
+import java.util.List;
+
+/**
+ * The Azure metric definition entries are of type MetricDefinition.
+ */
+public interface MetricDefinition {
```

Comment:
Should it extends `HasInner<MetricDefinitionInner>`?

Reply:
fixed.